app/views/pages/home.html.erb

```diff
@@ -18,5 +18,3 @@
 <% else %>
   <%= render partial: 'signed_in_empty' %>
 <% end %>
-
-<%= render partial: 'update_password_modal' %>
```

Comment:
I think we can add this back if we introduce [Devise Pwned Passwords](https://github.com/michaelbanfield/devise-pwned_password) to suggest to the user to change their password if it was found in a data breach. 

Otherwise, I'm removing this for now since there isn't a case for this anymore.

Reply:
Maybe we can comment this out with a TODO indicating this just so we don't forget!